assets/css/bootstrap-4.6.0.min.css

```
@@ -0,0 +1,7 @@
+/*# sourceMappingURL=bootstrap.min.css.map */
```

Minor, but I noticed my browser 404ing when trying to find the map file
```suggestion
/*# sourceMappingURL=bootstrap-4.6.0.min.css.map */
```